pkg/util/nsenter/nsenter.go

```diff
@@ -122,3 +122,8 @@ func (ne *Nsenter) SupportsSystemd() (string, bool) {
 	systemdRunPath, hasSystemd := ne.paths["systemd-run"]
 	return systemdRunPath, hasSystemd
 }
+
+// HostRootFsPath returns host rootfs mountpoint path in kubelet container.
+func (ne *Nsenter) HostRootFsPath() string {
```

Comment:
IMO there is no need to return such a constant variable in a function. You can just rename `hostRootFsPath` to `HostRootFsPath` if needed.